styles.css

```diff
@@ -0,0 +1,71 @@
+html,
+body {
+  height: 100%;
+}
+body {
+ margin-top: 15px;
+ color: #5A5A5A;
+ text-align: center;
+ font-size: 13px;
+ font-weight: 300;
+ font-family: "Open Sans", sans-serif;;
+
+}
+
+h1 {
+  font-weight: 300;
+  font-size: 28px;
+  margin-top: 10px;
+  margin-bottom: 0;
+
+}
+
+h3 {
+  font-weight: 600;
+  font-size: 13px;
+  margin: 0;
+}
+
+p {
+  margin-top: 2px;
+  margin-bottom: 20px;
+
+}
+.column {
+  width: 278px;
+  margin: 30px;
+  display: inline-block;
+  border: 2px solid #EDEDED;
+  border-radius: 10px;
+  padding: 0;
+}
+
+.container {
+  width: 282px;
+  height: 255px;
+  margin-left: -2px;
+  margin-top: -1px;
+  overflow: hidden;
+  padding: 0;
+}
+.container img {
+  width: 100%;
+  padding: 0;
+  -webkit-border-radius: 10px;
+  -moz-border-radius: 10px;
+  border-radius: 10px;
+}
+
+.bottom {
+  border-top: 2px solid #EDEDED;
+  width: 100%;
+}
+
+.figures {
+    display: inline-block;
+    margin-top: 0px;
+    margin-bottom: 0px;
+    margin-left: 40px;
+    margin-right: 40px;
```

Comment:
Here's some shorthand
```css
/** pseudo-code **/
margin: [top] [right] [bottom] [left]
```

* If you only included the top value, it will apply to all 4
* If you only included the top and right values, the top value applies to the top AND bottom; the right value applies to the right AND left
* If you only included the top and right and bottom values, the top value applies to itself, the right value applies to the right AND left, the bottom value applies to itself.
so...

```css
margin: 0 40px;
```